code/modules/client/preferences.dm

```diff
@@ -1682,13 +1682,13 @@ NOTE:  The change will take effect AFTER any current recruiting periods."}
 		<legend>Legend</legend>
 		<dl>
 			<dt>Never:</dt>
-			<dd>Automatically respond 'no' to all requests to become this role.</dd>
+			<dd>Decline this role for this round and all future rounds. You will not be polled again.</dd>
 			<dt>No:</dt>
-			<dd>Responds no to requests to become this role, with options to sign up to become it. Default.</dd>
```

Comment:
If we don't have any preference options that do this, I think we should.
Being able to see every role request, but only as opt-in, would be nice.